### PR TITLE
Update latest terms.

### PR DESCRIPTION
**Pull request by biocom**
# Summary

Updates the terms & conditions to the latest version.

### CODE REVIEW

File: src/custom/pages/TermsAndConditions/TermsAndConditions.md [hunk 1]
```diff
@@ -1,166 +1,462 @@
...
-<p><span style="text-decoration: underline;">FAILURES WITHIN DATA STORAGE SYSTEMS</span></p>
-<p>Our Platform may in part be established on servers at data centre facilities of third party providers and on&nbsp; distributed systems for storing and accessing data including IPFS. Where centralised services may be used, we may be required to transfer our Platform to different facilities, and may incur service interruption in connection with such relocation. Data centre facilities are vulnerable to force majeure events or other failures. Third party providers may suffer breaches of security and others may obtain unauthorised access to our server data. Where content is stored via distributed systems, there may be interference in&nbsp; content addressing, content linking, indexing and discovery. As techniques used to obtain unauthorised access change frequently and generally are not recognised until used against a target, it may not be possible to anticipate these techniques or to implement adequate preventive measures.</p>
+**CowSwap Terms & Conditions**
+
+<i><b>Last updated: July 2021</b></i>
```
Comment (alfetopito) [inline src/custom/pages/TermsAndConditions/TermsAndConditions.md:3]: August (release date)?

File: src/custom/pages/TermsAndConditions/TermsAndConditions.md [hunk 2]
```diff
@@ -1,166 +1,462 @@
...
+
+**WHAT YOU REQUIRE TO USE OUR PLATFORM**
+
+Users may use our Platform to interact with the Protocol and buy or sell
+ERC 20 tokens by submitting orders into a batch auction running on the
+Protocol. 
+
+A detailed step-by-step guide on how to use the Platform may be found
+[in the “FAQ” section on the
+Platform](https://cowswap.exchange/#/faq#what-is-cowswap).
```
Comment (alfetopito) [inline src/custom/pages/TermsAndConditions/TermsAndConditions.md:74]: Make it relative? For example, barn.cowswap.exchange will redirect to cowswap.exchange

**Review by alfetopito — APPROVED** [review]

File: src/custom/pages/TermsAndConditions/TermsAndConditions.md (hunk 2 above)
Comment (biocom) [reply src/custom/pages/TermsAndConditions/TermsAndConditions.md:74]: Good point, will change.

Comment (biocom) [reply src/custom/pages/TermsAndConditions/TermsAndConditions.md]: Updated now!

File: src/custom/pages/TermsAndConditions/TermsAndConditions.md (hunk 1 above)
Comment (biocom) [reply src/custom/pages/TermsAndConditions/TermsAndConditions.md]: Yes, updated :)

**Review by W3stside — APPROVED** [review]

**Comment by elena-zh** [comment]
Some cases: 
1. It would be nice to remove '1.' from the 'INFORMATION ON THE PLATFORM ARE NOT ADVICE' section
image
2. Links for forum.gnosis.io are missing
image
image
3. This link should be opened in a new tab (like it works in Barn env-t now)
image
4. It would be nice to add a navigation to APPENDIX (reported in #1115)
image
5. Personally, I'm not sure whether we need this (1) in the text 
image

File: src/custom/pages/TermsAndConditions/TermsAndConditions.md (hunk 2 above)
Comment (alfetopito) [reply src/custom/pages/TermsAndConditions/TermsAndConditions.md]: What do you think about opening this in the same page instead of a new one?

**Review by elena-zh — APPROVED** [review]
LGTM!
Just to notice: we have a low-priority enhancement on the board #1173 to make sections numbering in bold. I think it would be great to address this issue in this pr.

**Pull request merged**